### ISSUE TITLE
App Crashes when Backgrounded

### ISSUE THREAD

**Issue by jsm** [wix/react-native-keyboard-input#20]
Hey Guys,

Running into an issue with this crashing our app on some phones when the app is backgrounded.
calls getWindow(), which returns null, since it's backgrounded, causing a null pointer references, crashing the app.

```
FRAME	MODULE	CRASH DETAILS
0	com​.wix​.reactnativekeyboardinput	
ViewUtils.java ​ Line 15
ViewUtils.getWindow
1	com​.wix​.reactnativekeyboardinput	
ReactSoftKeyboardMonitor.java ​ Line 136
ReactSoftKeyboardMonitor.removeAllLayoutListeners
2	com​.wix​.reactnativekeyboardinput	
ReactSoftKeyboardMonitor.java ​ Line 98
ReactSoftKeyboardMonitor.onHostDestroy
3	com​.facebook​.react​.bridge	
ReactContext.java ​ Line 236
ReactContext.onHostDestroy
4	com​.facebook​.react	
ReactInstanceManager.java ​ Line 651
ReactInstanceManager.moveToBeforeCreateLifecycleState
5	com​.facebook​.react	
ReactInstanceManager.java ​ Line 575
ReactInstanceManager.onHostDestroy
6	com​.facebook​.react	
ReactInstanceManager.java ​ Line 588
ReactInstanceManager.onHostDestroy
7	com​.facebook​.react	
ReactActivityDelegate.java ​ Line 142
ReactActivityDelegate.onDestroy
8	com​.facebook​.react	
ReactActivity.java ​ Line 72
ReactActivity.onDestroy
9	android​.app	
Activity.java ​ Line 6471
Activity.performDestroy
10	android​.app	
Instrumentation.java ​ Line 1149
Instrumentation.callActivityOnDestroy
11	android​.app	
ActivityThread.java ​ Line 4063
ActivityThread.performDestroyActivity
```

![image](https://cloud.githubusercontent.com/assets/1446054/26079962/973bd02a-3992-11e7-8693-d9bf9d86a071.png)

**Comment by artald** [comment]
Thanks @jsm !

@d4vidi is it a know issue?

**Comment by d4vidi** [comment]
Nope. I bet it's new, introduced due to recent navigation support implementation.

**Comment by d4vidi** [comment]
@jsm thanks for this submission! The issue btw is with the activity (or the whole app) going down. I'll work it out asap.

**Comment by d4vidi** [comment]
Should be fixed in 3.0.18.

**Comment by jsm** [comment]
@d4vidi thanks a bunch!
